### PR TITLE
Restore a couple of convenience methods

### PR DESCRIPTION
**Pull request by andresrc**
@reviewbybees

### CODE REVIEW

**Comment by amuniz** [comment]
Restore? I'd say _add_, that methods never existed AFAICT

:bee:

**Comment by andresrc** [comment]
Well, the removed ones were more specific:

https://github.com/jenkinsci/jenkins-test-harness/commit/90580fffda778ee8eee4f492ac85297d42c8ca7f#diff-686db5e78025a85e9ed2a488aa6d34fcL749

I'm very open-minded :smile:

**Comment by andresrc** [comment]
@reviewbybees done

**Comment by andresrc** [comment]
No more comments in more than 3 weeks, merging....

File: .gitignore [hunk 1]
```diff
@@ -1 +1,15 @@
 target
+work
```
Comment (jglick) [inline .gitignore]: Why? This is not a plugin.

Comment (andresrc) [inline .gitignore]: eeehh, blind copy from the ignores of another project :(

**Pull request merged**